docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
 ---
-title: Creating custom Expectations
+title: Overview
 ---
-import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Prerequisites from './components/prerequisites.jsx'
```

Comment:
It seems this could be a more broadly shared component rather than being local to create_custom_expectations

Reply:
Agreed. I actually copied it from `docs/guides/connecting_to_your_data/components/prerequisites.jsx`.

There are a few different versions of this already. AFACIT, they differ only in the defaults that they include. These defaults tend to differ by area within the ToC, so it makes sense to have them split out by subdirectory in the docs.

I'd suggest doing a separate PR to refactor all these `Prerequisites` components to use a common base class.